src/shared/firewall-util-nft.c

```diff
@@ -964,6 +967,46 @@ static int fw_nftables_add_masquerade_internal(
                                               source, source_prefixlen);
 }
 
+int nft_set_element_modify_in_addr_open(
+                FirewallContext **ctx,
+                bool add,
+                NFTSet *nft_set,
+                int af,
+                const union in_addr_union *address,
+                unsigned int prefixlen) {
+        const char *table, *set;
```

Comment:
Please insert an empty line between function arguments and variable declarations.

Reply:
ok